specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json

```diff
@@ -221,93 +261,6 @@
           }
         }
       }
-    },
-    "/subscriptions/{subscriptionId}/providers/microsoft.insights/workbooks/{resourceName}": {  
-      "get": {
-        "description": "Get a single workbook by its resourceName.",
-        "operationId": "Workbook_Get",
-        "parameters": [
-          {
-            "$ref": "#/parameters/SubscriptionIdParameter"
-          },
-          {
-            "$ref": "#/parameters/WorkbookResourceNameParameter"
-          },
-          {
-            "$ref": "#/parameters/LocationParameter"
-          },
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "A workbook definition.",
-            "schema": {
-                "$ref": "#/definitions/Workbook"
-            }
-          },
-          "default": {
-            "description": "Error response describing why the operation failed.",
-            "schema": {
-              "$ref": "#/definitions/WorkbookError"
-            }
-          }
-        },
-        "x-ms-examples": {
-          "WorkbookGet": {
-            "$ref": "./examples/WorkbookGet.json"
-          }
-        }
-      }
-    },
-    "/subscriptions/{subscriptionId}/providers/microsoft.insights/workbooks": {  
```

Comment:
It's a breaking change to remove an operation here. No breaking changes should be made to an existing stable Swagger.

Reply:
@jianghaolu Thanks for reviewing this. This is new resource type that we just got approved last week. This is not deployed to PROD yet. This is the endpoint we overlooked and needed to be removed.

Reply:
@ericc1103 Sounds good. When are you deploying to PROD?

Reply:
I am currently working with ARM team and ETA is tomorrow.

Reply:
@ericc1103 Please add "merge on [date]" on the PR title once you have an ETA. I've approved for now and on the deployment date we can merge.

Reply:
@jianghaolu It is now merged in PROD ARM. You can go ahead and approve to merge.

Reply:
Merging now.